add openEO specific stac info (#1)
Open-EO/openeo-stac-extensions: README.md

# openeo-stac-extensions

> Key-values within STAC Extensions specific to openEO

Reviewer: ```suggestion
Additional fields for STAC used within openEO and other details about how existing STAC extensions are used in openEO.
```

## Datacube Extensions

### Horizontal Spatial Dimensions Object

> - reference_system object with `"id":{"authority":"OGC","version":"1.3","code":"Auto42001"}`: Deepending on the location a different default crs (in this case the corresponding UTM zone) is used. For more info see `urn:ogc:def:crs:OGC:1.3:AUTO42001:99:8888` definded by OGC ([Definition identifier URNs in OGC namespace](https://portal.ogc.org/files/?artifact_id=24045))

Reviewer: ```suggestion
- `reference_system` object with `"id":{"authority":"OGC","version":"1.3","code":"Auto42001"}`: Deepending on the location a different default crs (in this case the corresponding UTM zone) is used. For more info see `urn:ogc:def:crs:OGC:1.3:AUTO42001:99:8888` definded by OGC ([Definition identifier URNs in OGC namespace](https://portal.ogc.org/files/?artifact_id=24045))
```

> ## Electro-Optical Extension 

Reviewer: ```suggestion
## Electro-Optical / Raster Extension 
```

> ### eo:bands

Reviewer: ```suggestion
### eo:bands / raster:bands
```

> | Field name | Value | Description |

Reviewer: ```suggestion
| Field name | Data Type | Description |
```

| -----------|-------|-------------|
| openeo:gsd | [GSD Object](#GSD-Object)| An array containing GSD value and unit. |

> 

Reviewer: ```suggestion

* For EO data the field is added to `eo:bands` entries.
* For SAR and non-optical data the field is added to `raster:bands` entries.

```

> ### GSD Object

Reviewer: ```suggestion
#### GSD Object
```

> | Field name | Value | Description |

Reviewer: ```suggestion
| Field name | Data Type | Description |
```

| -----------|----------|-------------|
| value      | [number] | **REQUIRED.** Collection's default pixel spacing in [x/y] direction in the unit of the reference_system specified in the datacube extension. Conversion factor between m and degrre (111 km = 1°) |

> | unit       | m/°    | **REQUIRED.** Specifies the unit of the value field within the GSD object. |

Reviewer: ```suggestion
| unit       | string    | **REQUIRED.** Specifies the unit of the value field within the GSD object. One of `m` (meters) or `°` (degrees). |
```

> *For sar and non optical data openeo:gsd is placed within `raster:bands`.

Reviewer: ```suggestion
```
